src/components/SearchPlugin.js

```diff
@@ -102,11 +104,28 @@ class App extends Component {
         const { openWithModal } = this.props;
         const isValid = appname && credentials;
         const isOpenWithModal = Boolean(openWithModal);
+        let fontFamilyLink = '';
+        if (
+            theme.typography &&
+            theme.typography.fontFamily &&
+            theme.typography.fontFamily !== 'default'
+        ) {
```

Comment:
Use get from lodash

Reply:
We currently dont have lodash installed and the app uses conditional check I can create a separate PR to use lodash everywhere.

Reply:
Sure (y)